### DISCUSSION TITLE
[Feature request]: Official Docker Support

### DISCUSSION THREAD

**Discussion by SpamMusubi153** [Shelf-nu/shelf.nu#1294] [Q&A]
### Is your feature request related to a problem? Please describe?

Hello Shelf team!
It would be really nice if Shelf could eventually provide official self-hosting Docker support.

### Describe the solution you'd like

A lot of the people and places I've been helping (especially teachers that share a lot of their own film equipment with their students) work at too small a scale to afford official hosting and don't have a dedicated machine they can spare to just run a Shelf environment (they usually need to run a few things, and keeping them in containers has really helped me with overall stability).

If Shelf were able to provide official Docker support I'd feel a lot more comfortable recommending, installing, and maintaining it on their machines. Shelf looks like such as great project that could really help them out, and once they start using it I'm sure they'd want to find a way to give back to the effort too.

Thanks for the great work.

**Comment by DonKoko** [comment]
Hey @SpamMusubi153 . Thanks for opening an issue and sharing your ideas/request.

We are not planning to officially support docker in the near future. 
Our self-hosted version of shelf is meant for people who are knowledgeable enough to set it up themselves, with the provided documentation, whether they choose to use docker or not. For people without this knowledge, we have a free tier of shelf with minor limitations.

I have converted your issue into a discussion for future reference in our community.

**Reply by SpamMusubi153** [reply]
Thanks for the reply. I understand that the team needs a way to make money to support development.
Just to keep in mind for the future, if you eventually feel comfortable doing this, I think it could be an option to build underlying community support.
